.github/workflows/postgis_online.yml

```diff
@@ -44,7 +44,18 @@ jobs:
         database=gttest
         driver=org.postgresql.Driver
         EOT
+        cat <<EOT >>   ~/.geotools/postgis_datastore.properties
+        user=geotools
+        port=5432
+        password=geotools
+        passwd=geotools
+        url=jdbc\:postgresql\://localhost/gtindexmosaic
```

Comment:
Where is this "gtindexmosaic" database created? It could just use the same database as the other tests, the chance of table conflict should be very slim, otherwise the separate database must be created (not sure if a call to createdb is available from here, would require a bit more research).